roles/inspec/tasks/main.yml

```diff
@@ -10,18 +10,39 @@
     mode: 0755
   with_items:
     - /etc/inspec
-    - /etc/inspec/controls
-    - /etc/inspec/libraries
+    - /etc/inspec/host-controls
+    - /etc/inspec/host-controls/controls
+    - /etc/inspec/host-controls/libraries
 
-- name: inspec config file
+- name: download and unpackage dependency profiles
+  unarchive:
+    copy: no
+    src: '{{ item }}'
+    dest: /etc/inspec/
+    remote_src: True
+  with_items: '{{ inspec.dependency_urls }}'
+
+- name: inspec profile yml
+  template:
+    src: etc/inspec/host-controls/inspec.yml
+    dest: /etc/inspec/host-controls/inspec.yml
+    mode: 0644
+
+- name: inspec control file
+  template:
+    src: etc/inspec/host-controls/controls/host.rb
+    dest: /etc/inspec/host-controls/controls/host.rb
+    mode: 0644
+
+- name: inspec attributes
   template:
-    src: etc/inspec/inspec.yml
-    dest: /etc/inspec/inspec.yml
+    src: etc/inspec/host-controls/attributes.yml
```

Comment:
because we're doing blockinfile to modify this file, if we lay it down as a template like this we lose idempotency.   I think using copy: you can tell it to only do it if the file doesn't already exist.